app/src/test/java/io/hstream/testing/ClusterKillNodeTest.java

```diff
@@ -46,18 +68,40 @@ public void teardown() throws Exception {
   }
 
   @Test
-  void listStreamAfterKillNode() {
+  void listStreamAfterKillNodes() {
+    String stream = randStream(hStreamClient);
     hServers.get(0).close();
-    hStreamClient.listStreams();
+    hServers.get(1).close();
+    Assertions.assertEquals(stream, hStreamClient.listStreams().get(0).getStreamName());
+  }
+
+  @Test
+  void killAllShouldFail() throws Exception {
+    for (int i = 0; i < 3; i++) {
+      terminateHServerWithLogs(0, i);
+    }
+    Assertions.assertThrows(
+        Exception.class,
+        () -> {
+          hStreamClient.listStreams();
+        });
   }
```

Comment:
The test name is misleading. It reads like we can not kill all nodes, or something will fail. But you just want to say the client will fail to perform any operation if all nodes are killed.

Reply:
`logMsgPathPrefix` now